.deploy/naiserator.yaml

```diff
@@ -54,6 +54,11 @@ spec:
       destinations:
         - id: "grafana-lgtm"
         - id: "elastic-apm"
+    logging:
+      destinations:
+        - id: elastic
+        - id: loki
+        - id: secure_logs
```

Comment:
Er dette den gamle? Heter ikke den nye team_logs?

Reply:
Det er den gamle ja - jeg er usikker om den trenges eller ikke så jeg bare tok den med.

Reply:
Ja, det var den gamle det ja .)

Reply:
Skal en ta i bruk team_logs?

Reply:
Tror det finnes ikke en destination som heter team_logs. Men team logs virker allerede for fp-sak. Sjekk i logback.xml og den commiten der.

Reply:
Ser at team_logs er en mulig destinations: https://docs.nais.io/observability/logging/reference/destinations/?h=logging#destinations

Reply:
Jeg følgte denne her https://docs.nais.io/observability/logging/how-to/team-logs/ og ting virker helt ok i fp-formidling. Mulig vi kan droppe secure_logs også. Tror at den ikke trenges

Reply:
Vi prøver :D